### PR TITLE
feat: display entity API

### PR DESCRIPTION
**Pull request by zlataovce**
Supersedes #267.

TODO:
 - [x] add `BlockDisplay` example
 - [x] add transformation example
 - [x] add interpolation example
 - [x] explain some more `Display` methods
 - [x] link to minecraft.wiki article (https://minecraft.wiki/w/Display)
 - [x] mention hiding the entities from other players - the point of the superseded PR

### CODE REVIEW

**Comment by MiniDigger** [comment]
did you change the url for that one page? if so, it needs a redirect
(cant really tell on mobile)

**Comment by zlataovce** [comment]
> did you change the url for that one page? if so, it needs a redirect (cant really tell on mobile)

the slug is the same, only thing that changed was the file path

File: docs/paper/dev/api/entity-api/entity-teleport.mdx [hunk 1]
```diff
@@ -3,7 +3,7 @@ slug: /dev/entity-teleport
 description: The entity teleportation API and how to use it.
 ---
 
-# Entity Teleportation
+# Teleportation
 
 Entities can be instantaneously teleported to specific positions, synchronously and asynchronously with the
```
Comment (MiniDigger) [inline docs/paper/dev/api/entity-api/entity-teleport.mdx:8]: first time seeing this page and idk if in scope, but we really should explain when and why to use the async methods (i.e. no sync chunk loads, performance 📈)

File: docs/paper/dev/api/entity-api/display-entities.mdx [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+
+Added in 1.19.4, [display entities](https://minecraft.wiki/w/Display) are a way to display various
+things in the world, like blocks, items and text.
+
+By default, these entities have no hitbox, don't move, make sounds or take damage,
+making them the perfect for all kinds of applications, like holograms.
+
+## Types
+
+:::tip[Decorative displays]
```
Comment (MiniDigger) [inline docs/paper/dev/api/entity-api/display-entities.mdx:16]: I feel like this should be under a "useful use cases" section at the bottom, after the basics are explained

File: docs/paper/dev/api/entity-api/entity-teleport.mdx (hunk 1 above)
Comment (zlataovce) [reply docs/paper/dev/api/entity-api/entity-teleport.mdx:8]: that's out of scope for this PR, but I could add that in a later one

**Review by olijeffers0n — APPROVED** [review]

**Pull request merged**